ietf/templates/person/merge.html

```diff
@@ -1,5 +1,5 @@
+{# Copyright The IETF Trust 2025, All Rights Reserved #}
```

Comment:
Usual practice is to preserve the original year (i.e., `2015-2025`)

Reply:
Fixed in commit 5e69a36a4e4655ba6994400a0cb32c97e70a6ea5